packages/ui-components/src/components/advance-date-select-dropdown/advance-date-select-dropdown.types.ts

```diff
@@ -0,0 +1,24 @@
+import { ComputePositionConfig } from '@floating-ui/dom';
+import { EventEmitter } from '@stencil/core';
+import { ITextField, ICalendarAdvanceTimeChange, ICalendarAdvanceDateSelector, ICalendarAdvanceDateSelectorEvents } from '../../types';
+
+export interface IAdvanceDateSelectDropdown extends ICalendarAdvanceDateSelector {
+	/** (optional) The text field dropdown custom configurations */
+	inputConfig?: Partial<ITextField>;
+	/** (optional) The text field date format mask */
```

Comment:
what kind of date format mask standard are we using? I suspect it will use the same as standard js dates (ISO 8601)

Reply:
You're right, will update 💪